doc/api/process.md

```diff
@@ -1188,6 +1188,15 @@ console.log(process.getgroups());         // [ 27, 30, 46, 1000 ]
 *Note*: This function is only available on POSIX platforms (i.e. not Windows
 or Android).
 
+## process.isExiting
+<!-- YAML
+added: vX.X.X
+-->
+
+This read-only boolean tells whether the process is currently exiting. That is,
+when the `exit` even has been fired and no further scheduled asynchronus code
```

Comment:
1. even -> event?
2. Maybe it is worth to add a type before the description for consistency with other properties:
```
* {boolean}
```